lib/linter/index.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
```

Comment:
I'm not seeing it was used somewhere. (plz let me know if I'm missing something)
what about `lib/linter/linter.js` => `lib/linter/index.js`?

Reply:
This file is used for the directory path such as `../linter`. I have avoided writing implementation into `index.js` because tab editor may show only the basename on their tabs (I.e., avoiding to list many `index.js`).